ReactiveCocoa/Swift/Bag.swift

```diff
@@ -60,7 +60,7 @@ internal struct Bag<T> {
 	/// will reset all current identifiers to reclaim a contiguous set of
 	/// available identifiers for the future.
 	private mutating func reindex() {
-		for var i = 0; i < elements.count; i++ {
+		for i in 0..<elements.count {
```

Comment:
It's only fine because it's an array, though `indices` would be more correct.
But sure, this is okay.